frontend/views/containers/TimeTravel.vue

```diff
@@ -66,12 +62,12 @@ export default {
       },
       config: {
         sliderConfig: {
-          tooltip: 'focus',
           marks: true,
           adsorb: true,
-          lazy: true,
+          lazy: false,
```

Comment:
I believe it's false by default, so we can remove this.

Reply:
Well, just in case the default changes, or even if it doesn't to remind us of its purpose, I think it's OK leaving it there.